pacta/cj_build_tag.sh

```diff
@@ -0,0 +1,58 @@
+#! /bin/bash
+
+tag="${1:-latest}"
+clones="${2:-PACTA_analysis create_interactive_report StressTestingModelDev pacta-data}"
+url="git@github.com:2DegreesInvesting/"
+
+for repo in ${clones}
+do
+    remote="${url}${repo}.git"
+    git clone -b master ${remote} --depth 1
+    echo "--"
+done
+
+for repo in ${clones}
+do
+    echo "${repo} HEAD sha:"
+    git -C "${repo}" rev-parse HEAD
+    echo "--"
+done
+
+for repo in ${clones}
+do
+    git -C "${repo}" tag -a "${tag}" -m "Release pacta ${tag}" HEAD
+    echo "${repo}"
+    echo "$(git -C ${repo} log --pretty='%h %d <%an> (%cr)' | head -n 1)"
+    echo "--"
+done
+
+docker rmi 2dii_pacta
+docker build ./ --tag 2dii_pacta:"${tag}"
```

Comment:
this isn't playing well on my machine.... if I don't use an argument with the shell script command and there's no existing 2dii_pacta in my docker system, this loads a new image in my docker system with no name and spins up a zombie docker instance of that image which is a bit difficult to get rid of... so I'm currently removing the `:"${tag}"` bit of this on my local copy

Reply:
Oh — I’ll look into that

Reply:
I did a few experiments and I'm not able to reproduce the problem. This is surprising because I'm pretty sure I saw one of those `<none>` images recently. If you are able to reproduce, it'd be good to share screen or document more precisely how you got there.

Reply:
Now that I've used this numerous times in the past days and understand better the tagging thing, I don't see any real reason to use this script without setting the tag, so this problem I was having before is kind of irrelevant.